library/seq.lisp

```diff
@@ -562,6 +562,11 @@ It attempts to rebalance with a minimum of array copying."
                           (cl:rest leaf-arrays)
                           :initial-value `(LeafArray ,(cl:funcall make-ary (cl:first leaf-arrays))))))))))
 
+(cl:defmethod cl:print-object :around ((self seq) stream)
+  (cl:format stream "#.(SEQ~{ ~A~})"
```

Comment:
I would print this as an unreadable object since it won't be readable outside of a Coalton context (unlike the other examples).

    (defmethod ...
     (print-unreadable-object (... :type nil :identity nil)
        (prin1 'seq) ; print package as necessary
        (format stream "~{ ..." ...)))

Reply:
`define-type` does not implement the `print-object` method for the `seq` class, it implements it for both subclasses LeafArray and RelaxedNode, if I just define the method for the `seq` superclass, the "more specific" subclass method implementations are used.  There may be some CLOS trick I don't know about to make it use the more generic method here.

I will make it print unreadably, but why is `#.(CELL 1)` supposed to work?  I can't get that to do anything valid in my REPL in either CL or Coalton.  Also would using the originally suggested `#.(SEQ:MAKE 1 2 3)` satisfy the readability issue?